cmd/subcommands/command.go

```diff
@@ -0,0 +1,89 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/harmony-one/go-sdk/pkg/console"
+	"github.com/mitchellh/go-homedir"
+	"github.com/spf13/cobra"
+	"log"
+	"os"
+	"path"
+)
+
+func init() {
+	shardNum := 0
+	net := "mainnet"
+
+	cmdCommand := &cobra.Command{
+		Use:   "command",
+		Short: "Start an interactive JavaScript environment (connect to node)",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return openConsole(net, shardNum)
+		},
+	}
+
+	cmdCommand.Flags().IntVar(&shardNum, "shard", 0, "used shard(default: 0)")
+	cmdCommand.Flags().StringVar(&net, "net", "mainnet", "used net(default: mainnet, choose: mainnet or testnet)")
+
+	RootCmd.AddCommand(cmdCommand)
+}
+
+func checkAndMakeDirIfNeeded() string {
+	userDir, _ := homedir.Dir()
+	hmyCLIDir := path.Join(userDir, ".hmy_cli", "command")
```

Comment:
use defined constants from common: https://github.com/harmony-one/go-sdk/blob/master/pkg/store/local.go#L21 instead of values.